.github/workflows/update-kernels.yml

```diff
@@ -21,13 +44,31 @@ jobs:
         with:
           ref: kernels
 
+      - name: Setup jq
+        run: apt-get update && apt-get install jq -y
+
       - name: Run crawler for x86_64
         run: |
-          kernel-crawler crawl --distro=* > x86_64/list.json
+          mkdir x86_64_tmp
+          kernel-crawler crawl --distro=${{ env.DISTRO }} > x86_64_tmp/list.json
 
       - name: Run crawler for aarch64
         run: |
-          kernel-crawler crawl --distro=* --arch=aarch64 > aarch64/list.json
+          mkdir aarch64_tmp
+          kernel-crawler crawl --distro=${{ env.DISTRO }} --arch=aarch64 > aarch64_tmp/list.json
+
+      - name: Single distro update
+        if: github.event_name == 'workflow_dispatch' && env.DISTRO != '*'
+        run: |
+          jq --arg distroKey "${{ env.DISTRO }}" --slurpfile newValues x86_64_tmp/list.json \
```

Comment:
I am not sure about this: i mean, i don't see where we are using the actual real complete json (ie: the one inside our repositoriy: `$arch/list.json`. Perhaps some typos here?